install/kubernetes/cilium/values.yaml

```diff
@@ -895,6 +895,15 @@ keepDeprecatedLabels: false
 # -- Keep the deprecated probes when deploying Cilium DaemonSet
 keepDeprecatedProbes: false
 
+# -- fine tune the liveness, readiness and startup probes
+# 105 translates to 120s (initial delay) + 3 x 30s of the old readiness probe
+startupProbeFailureThreshold: 105
+startupProbePeriodSeconds: 2
+livenessProbeFailureThreshold: 10
+livenessProbePeriodSeconds: 30
+readinessProbeFailureThreshold: 3
+readinessProbePeriodSeconds: 30
+
```

Comment:
Just a small nit, would it look better if it was something like:
```
startupProbe:
  failureThreshold: 105
  periodSeconds: 2
```

Reply:
Sure, I'll remodel and force-push.